libs/execution_context/ebpf_program.h

```diff
@@ -475,6 +475,44 @@ extern "C"
     void
     ebpf_program_set_flags(_Inout_ ebpf_program_t* program, uint64_t flags);
 
+    /**
+     * @brief Set the context_descriptor in the program context header.
+     *  Writes a pointer to the ebpf_program_t object to Slot [1].
+     *
+     *  @note Extension must support context headers.
+     *
+     * @param[in] context_descriptor Pointer to the context descriptor for the program.
+     * @param[in,out] program_context Pointer to the program context to set the header for.
+     */
+    void
+    ebpf_program_set_header_context_descriptor(
+        _In_ const ebpf_context_descriptor_t* context_descriptor, _Inout_ void* program_context);
+
+    /**
+     * @brief Get the context descriptor from the program context header.
+     *  Slot [1] contains the context descriptor pointer.
+     *
+     *  @note Extension must support context headers.
+     *
+     * @param[in] program_context Pointer to the program context.
+     * @param[out] context_descriptor Pointer to the program context to set.
+     */
+    void
+    ebpf_program_get_header_context_descriptor(
+        _In_ const void* program_context, _Outptr_ const ebpf_context_descriptor_t** context_descriptor);
+
+    /**
+     * @brief Get the data start and end pointers from the program context.
+     *
+     *  @note Extension must support context headers.
```

Comment:
```suggestion
     * @note Extension must support context headers.
```
Make doxygen line up.  Also in line 495 and perhaps elsewhere.